ranges/subset.js

```diff
@@ -3,6 +3,7 @@ const Comparator = require('../classes/comparator.js')
 const { ANY } = Comparator
 const satisfies = require('../functions/satisfies.js')
 const compare = require('../functions/compare.js')
+const parseOptions = require('../internal/parse-options.js')
```

Comment:
```sh
$ grep -r parse-options .|grep require|grep parse-options.js
./ranges/simplify.js:const parseOptions = require('../internal/parse-options.js')
./ranges/subset.js:const parseOptions = require('../internal/parse-options.js')
./test/internal/parse-options.js:const parseOptions = require('../../internal/parse-options.js')
```

Everywhere else requires `internal/parse-options` without `.js`

Reply:
I just keep the same pattern as other imports, I think we could leave it as it is and have another PR to standardize.

Because doesn't make too much sense to keep just parse-options without `.js` but is out of the scope to remove all imports from `.js`.

Reply:
I'll be happy to make a separate PR removing the extension from all the imports :-)